src/AnonymousUser/AnonymousUser.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>

```

Comment:
How about upping to netstandard2.1, this way we could target a more recent ASP.NET Core